DSA-Campaign-Uplift-Estimation/src/main/webapp/Resources/NavBar/navbar_script.js

```diff
@@ -36,7 +36,7 @@ function getLogsStatus() {
         // login link should skip login and head into home. If the user is
         // not on the login screen then the url will be to sign out.
         if(logStatus.isLoggedIn) {
-            if(window.location.pathname === "/index.html"){
+            if(window.location.pathname === "/index.html" || window.location.pathname === "/"){
                 console.log('User is Logged In');
                 link.href = '../Home/home.html';
```

Comment:
What's the point of line 41 link.href = '../Home/home.html'; ? Don't you want to do link.href = logStatus.Url; regardless if you are logged in?

Reply:
No, because Adithya didn't want code repetition when it came to handling logging and logging out.  He wanted a single function that can handle both logging in and out. So when your logged in, the link is always going to be set to log out. When your logged out the link is always going to be set to log in.

Reply:
Yeah, so that's pretty much what I was saying - when you're logged in, the link should always be set to log out, so why are you doing link.href = '../Home/home.html'; ?

Reply:
Ohhh, I think you meant to change the window location instead of changing link.href.